src/cdsetool/download.py

```diff
@@ -55,9 +69,12 @@ def download_feature(feature, path, options=None):
             for chunk in response.iter_content(chunk_size=1024 * 1024 * 5):
                 file.write(chunk)
                 status.add_progress(len(chunk))
-
-        shutil.move(tmp, result_path)
-
+        valid_checksum = validity_check(tmp, feature)
+        if valid_checksum in (Validity.VALID, Validity.IGNORE):
```

Comment:
I have no idea if `not in` works, but you can save an intermediate variable and indentation by inverting the condition:

```python
if validity_check(tmp, feature) not in (Validity.VALID, Validity.IGNORE):
    log.error(f"Faulty checksum for {filename}")
    os.remove(tmp)
    return None
```